lib/CampMinder/EstablishConnection.rb

```diff
@@ -39,17 +38,17 @@ def connect
 
     http.use_ssl = true
     request = Net::HTTP::Post.new(uri.request_uri)
-    request.set_form_data({'fn' => 'EstablishConnection', 'businessPartnerID' => CampMinder::BUSINESS_PARTNER_ID, 'signedObject' => signed_object})
+    request.set_form_data({"fn" => "EstablishConnection", "businessPartnerID" => CampMinder::BUSINESS_PARTNER_ID, "signedObject" => signed_object})
```

Comment:
Line is too long. [147/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.